spec/services/staff_nomis_checker_spec.rb

```diff
@@ -175,7 +179,18 @@
     describe '#errors_for' do
       let(:slot) { visit.slots.first }
 
+      # TODO: Cleanup after global test config for feature flags has
+      # been removed from cong/test.rb
```

Comment:
I would not not know what to here. Can you be a bit more explicit?
also I believe you meant to write `config/environments/test.rb` ?